lib/text_search.rb

```diff
@@ -5,4 +5,13 @@ def lemmatize(word)
     lem = Lemmatizer.new
     lem.lemma(word)
   end
+
+  def non_hyphenate_query(word)
+  	word.gsub('-','')
```

Comment:
Would you mind using two-space indents here? It helps us keep the code tidy!

Reply:
Did that. Thank you!